### PR TITLE
Tentative new file structure for module_api

### PR DESCRIPTION
**Pull request by bigbitbus**
# Description
New directory/file structure of opta repo for module-api changes.

# Safety checklist
* [x] This change is backwards compatible and safe to apply by existing users
* [x] This change will NOT lead to data loss
* [x] This change will NOT lead to downtime who already has an env/service setup

## How has this change been tested, beside unit tests?
YOUR_ANSWER

### CODE REVIEW

File: modules/datadog/datadog_logging.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import os
```
Comment (NitinAgg) [inline modules/datadog/datadog_logging.py:1]: this isn't a module

**Review by NitinAgg — COMMENTED** [review]
I think the datadog logging changes creeped in here. but otherwise looks good

File: modules/datadog/datadog_logging.py (hunk 1 above)
Comment (bigbitbus) [reply modules/datadog/datadog_logging.py]: Good catch, fixed.

**Review by NitinAgg — APPROVED** [review]

**Comment by bigbitbus** [comment]
Lot of changes, for now registry yaml is still picked up from config/

**Pull request merged**